docs/references/protocol/transactions/types/ledgerstatefix.md

```diff
@@ -0,0 +1,68 @@
+---
+seo:
+    description: Repair corruptions to the XRP ledger.
+labels:
+  - Utilities, troubleshooting
+---
+# LedgerStateFix
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/LedgerStateFix.cpp "Source")
+
+
+`LedgerStateFix` is a general purpose transaction used to fix specific issues affecting the XRP ledger. You submit the transaction with the `LedgerFixType` value set to indicate the particular  error state to correct.
+
+## Example {% $frontmatter.seo.title %} JSON
+
+```json
+{
+   "Account" : "<Signer and fee payer>",
+   "Fee" : "2000000",
+   "LedgerFixType" : 1,
+   "Owner" : "<Account with corrupted NFTokenPage directory",
+   "Sequence" : <n>,
+   "SigningPubKey" : "<Account's public key>",
+   "TransactionType" : "LedgerStateFix",
+   "TxnSignature" : "<Signature>",
+}
+```
+
+{% raw-partial file="/docs/_snippets/tx-fields-intro.md" /%}
+
+| Field | Data Type  | Required? | Description |
+|:------|:-----------|:----------|:------------|
+| `TransactionType` | uint16 | Required | Identifies this as a `LedgerStateFix` transaction. |
+| `Account` | STAccount | Required | Identifies the account signing and submitting the transaction as well as paying the Fee. |
```

Comment:
This is a common field and doesn't need to be listed in this table. Neither does `Fee`, nor `Flags`.